### PR TITLE
Add workflow db independent.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #15265, #18232, #18040

- [x] Fixed tests mysql 8.0.
- [x] Fixed tests pgsql 9, 10, 11, 12, 13.
- [x] Add workflow tests php7.4, mysql 5.7, mysql-lastets (8.0).
- [x] Add workflow tests php7.4, pgsql 10, 11, 12, 13.

### CODE REVIEW

**Comment by terabytesoftw** [comment]
@samdark Just add PHP 7.4, i don't know if it will be necessary to add all versions of PHP.

**Comment by terabytesoftw** [comment]
@samdark tests pgsql fails pgsql latest, 11.9

```php
1) yiiunit\framework\db\pgsql\ActiveRecordTest::testDefaultValues
Invalid characters passed for attempted conversion, these have been ignored

Problem: pgsql/Schema.php bindec() deprecated invalid string.

} elseif (preg_match("/^B'(.*?)'::/", $column->defaultValue, $matches)) {
    $column->defaultValue = bindec($matches[1]);
} elseif (strncasecmp($column->dbType, 'bit', 3) === 0 || strncasecmp($column->dbType, 'varbit', 6) === 0) {
    $column->defaultValue = bindec(trim($column->defaultValue, 'B\''));
```

File: framework/db/pgsql/Schema.php [hunk 1]
```diff
@@ -548,14 +548,21 @@ protected function findColumns($table)
...
+                    )
+                ) {
                     $column->defaultValue = new Expression($column->defaultValue);
                 } elseif ($column->type === 'boolean') {
                     $column->defaultValue = ($column->defaultValue === 'true');
                 } elseif (preg_match("/^B'(.*?)'::/", $column->defaultValue, $matches)) {
                     $column->defaultValue = bindec($matches[1]);
-                } elseif (strncasecmp($column->dbType, 'bit', 3) === 0 || strncasecmp($column->dbType, 'varbit', 6) === 0) {
-                    $column->defaultValue = bindec(trim($column->defaultValue, 'B\''));
+                } elseif (preg_match("#^'(\d+)'::\"bit\"$#", $column->defaultValue, $matches)) {
```
Comment (samdark) [inline framework/db/pgsql/Schema.php:564]:
```suggestion
                } elseif (preg_match("/^'(\d+)'::\"bit\"$/", $column->defaultValue, $matches)) {
```

**Review by samdark — COMMENTED** [review]
1. Need to skip MySQL and PostgreSQL tests from main test suite then.
2. Need changelog about the bug fixed for these tests to pass.

**Comment by terabytesoftw** [comment]
Done all mysql8 and postgres 9, 10, 11, 12, 13 tests pass in with validation of expression timestamp lowercase, uppercase. comment # https://github.com/yiisoft/yii2/issues/15265#issuecomment-675926912

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thank you!

**Pull request merged**